crates/bevy_sprite/src/render/mod.rs

```diff
@@ -173,7 +173,8 @@ impl SpecializedPipeline for SpritePipeline {
 
 #[derive(Component, Clone, Copy)]
 pub struct ExtractedSprite {
```

Comment:
Can I convince you to toss in some *very* basic doc strings for this struct as part of this PR too?